integration/dataservice-hosting-tests/tests-integration/tests/src/test/java/org/wso2/ei/dataservice/integration/test/odata/ODataSuperTenantUserTestCase.java

```diff
@@ -207,14 +214,18 @@ private static int sendPATCH(String endpoint, String content, String acceptType)
             httpPatch.setEntity(httpEntity);
         }
         HttpResponse httpResponse = httpClient.execute(httpPatch);
-        return httpResponse.getStatusLine().getStatusCode();
+        int statusCode = httpResponse.getStatusLine().getStatusCode();
+        httpPatch.releaseConnection();
+        return statusCode;
     }
 
     private static int sendDELETE(String endpoint, String acceptType) throws IOException {
         HttpClient httpClient = new DefaultHttpClient();
         HttpDelete httpDelete = new HttpDelete(endpoint);
         httpDelete.setHeader("Accept", acceptType);
         HttpResponse httpResponse = httpClient.execute(httpDelete);
-        return httpResponse.getStatusLine().getStatusCode();
+        int statusCode = httpResponse.getStatusLine().getStatusCode();
+        httpDelete.releaseConnection();
```

Comment:
Lets do the same as [1] here as well

[1] https://github.com/wso2/micro-integrator/pull/2831/commits/2ae5bc7c1272febd586fd8a900b2dbcf3788f59d#r1082429357